CHANGELOG.md

```diff
@@ -463,6 +463,9 @@ names are deprecated but still supported until the next major release.
 This release disables reporting out-of-memory events in debug mode, to reduce
 false positives.
 
+* Deliver each event in a separate request to avoid exceeding payload size limit
```

Comment:
Should there also be an entry in the v6 section for this PR?

Reply:
Added a note to the upgrade guide and moved this to v6 - the original PR was targeted at v5.